src/cmd/health.js

```diff
@@ -37,94 +47,315 @@ function print_usage() {
 }
 
 const HOSTNAME = "localhost";
-const PORT = 6443;
-const SERVICE = "nsfs";
+const NSFS_SERVICE = "nsfs";
+const RSYSLOG_SERVICE = "rsyslog";
+const health_errors = {
+  NSFS_SERVICE_FAILED: {
+        error_code: 'NSFS_SERVICE_FAILED',
+        error_message: 'NSFS service is not started properly, Please verify the service with status command.',
+    },
+    RSYSLOG_SERVICE_FAILED: {
+        error_code: 'RSYSLOG_SERVICE_FAILED',
+        error_message: 'RSYSLOG service is not started properly, Please verify the service with status command.',
+    },
+    NSFS_ENDPOINT_FAILED: {
+        error_code: 'NSFS_ENDPOINT_FAILED',
+        error_message: 'NSFS endpoint process is not running. Restart the endpoint process.',
+    },
+    NSFS_ENDPOINT_FORK_MISSING: {
+        error_code: 'NSFS_ENDPOINT_FORK_MISSING',
+        error_message: 'One or more endpoint fork is not started properly. Verify the total and missing fork count in response.',
+    },
+    STORAGE_NOT_EXIST: {
+      error_code: 'STORAGE_NOT_EXIST',
+      error_message: 'Storage path mentioned in schema pointing to the invalid directory.',
+    },
+    INVALID_CONFIG: {
+      error_code: 'INVALID_CONFIG',
+      error_message: 'Schema JSON is not valid, Please check the JSON format.',
+    },
+};
 
-async function nc_nsfs_health() {
+const fork_response_code = {
+  200: {
```

Comment:
I would change the keys to an informative key like RUNNING / MISSING_FORKS/ NOT_RUNNING so it can be used in the function itself, see the comment there